docs/modules/ROOT/pages/installing.adoc

```diff
@@ -209,3 +209,6 @@ The client will attempt to connect to your ownCloud server, and when it is succe
 * One to open your local folder.
 
 It will also start synchronizing your files.
+
+
+NOTE: Path to find the autostart settings for the windows desktop client: `Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run`
```

Comment:
What is the reaon / drive for this addition?
You already can set the autostart settings in the client...

![image](https://user-images.githubusercontent.com/3321281/102382547-dab26480-3fca-11eb-960c-f3882345526e.png)

Reply:
I think @voroyam just wanted to show where the settings to this are stored. He is on vacation this week. We can wait with the merge to be sure if you want.

Reply:
Just writing this in the way it is done is very unprofessional und confusing customers. If there are (very) special reasons for a particular audience, the reason and the audience must be declared. Please also see that this is a „Note“ and thererfore highlighted. Ordinary user DO NOT have any reason for using regedit and manipulating keys! In case this is a must have info, this must be rewritten and redesigned.
You can if you want make a proposal change which speeds up the process.